controller_interface/test/test_force_torque_sensor.cpp

```diff
@@ -214,31 +214,31 @@ TEST_F(ForceTorqueSensorTest, validate_all_custom_names)
   ASSERT_EQ(force_torque_sensor_->interface_names_[5], sensor_name_ + "/" + "torque.z");
 
   // assign values to force
-  hardware_interface::StateInterface force_x{
-    sensor_name_, fts_interface_names_[0], &force_values_[0]};
-  hardware_interface::StateInterface force_y{
-    sensor_name_, fts_interface_names_[1], &force_values_[1]};
-  hardware_interface::StateInterface force_z{
-    sensor_name_, fts_interface_names_[2], &force_values_[2]};
+  auto force_x = std::make_shared<hardware_interface::StateInterface>(
+    sensor_name_, fts_interface_names_[0], &force_values_[0]);
+  auto force_y = std::make_shared<hardware_interface::StateInterface>(
+    sensor_name_, fts_interface_names_[1], &force_values_[1]);
+  auto force_z = std::make_shared<hardware_interface::StateInterface>(
+    sensor_name_, fts_interface_names_[2], &force_values_[2]);
 
   // assign values to torque
-  hardware_interface::StateInterface torque_x{
-    sensor_name_, fts_interface_names_[3], &torque_values_[0]};
-  hardware_interface::StateInterface torque_y{
-    sensor_name_, fts_interface_names_[4], &torque_values_[1]};
-  hardware_interface::StateInterface torque_z{
-    sensor_name_, fts_interface_names_[5], &torque_values_[2]};
+  auto torque_x = std::make_shared<hardware_interface::StateInterface>(
+    sensor_name_, fts_interface_names_[3], &torque_values_[0]);
+  auto torque_y = std::make_shared<hardware_interface::StateInterface>(
+    sensor_name_, fts_interface_names_[4], &torque_values_[1]);
+  auto torque_z = std::make_shared<hardware_interface::StateInterface>(
+    sensor_name_, fts_interface_names_[5], &torque_values_[2]);
 
   // create local state interface vector
   std::vector<hardware_interface::LoanedStateInterface> temp_state_interfaces;
 
   // insert the interfaces in jumbled sequence
-  temp_state_interfaces.emplace_back(torque_y);
-  temp_state_interfaces.emplace_back(force_z);
-  temp_state_interfaces.emplace_back(force_x);
-  temp_state_interfaces.emplace_back(torque_z);
-  temp_state_interfaces.emplace_back(torque_x);
-  temp_state_interfaces.emplace_back(force_y);
+  temp_state_interfaces.emplace_back(torque_y, nullptr);
+  temp_state_interfaces.emplace_back(force_z, nullptr);
+  temp_state_interfaces.emplace_back(force_x, nullptr);
+  temp_state_interfaces.emplace_back(torque_z, nullptr);
+  temp_state_interfaces.emplace_back(torque_x, nullptr);
+  temp_state_interfaces.emplace_back(force_y, nullptr);
```

Comment:
```suggestion
  temp_state_interfaces.emplace_back(torque_y, nullptr);
  temp_state_interfaces.emplace_back(force_z);
  temp_state_interfaces.emplace_back(force_x);
  temp_state_interfaces.emplace_back(torque_z);
  temp_state_interfaces.emplace_back(torque_x);
  temp_state_interfaces.emplace_back(force_y);
```
We don't need to define the nullptr right?, we already have a constructor doing pretty similar

Reply:
no, there is no such constructor. should I add it?